src/server/service/agent.ts

```diff
@@ -56,6 +69,12 @@ export class Agent {
 
     const message = await this.requestMessageFromLLM();
 
+    // If inference failed or bot decided not to reply, let the agent be silent
+    if (!!!message || message.includes("001001")) {
```

Comment:
You commended out the "001001" response in the prompt do you still want to keep it here?

Reply:
That's coming in the next PR, so no big deal